ParseUI-Widget/build.gradle

```diff
@@ -27,6 +27,7 @@ group = 'com.parse'
 
 dependencies {
     compile 'com.parse:parse-android:1.10.3'
+    compile 'com.android.support:support-v13:21.0.3'
```

Comment:
Most of the projects I was working on were using native Fragments, so I went with the v13 support library.  However, I can easily make changes to this to use the v4 support library, or I can have 2 different versions of this adapter to ease development for those who have implemented v4 or native Fragments.